src/main/ml-modules/root/com.marklogic.smart-mastering/survivorship/merging/base.xqy

```diff
@@ -1558,9 +1636,6 @@ declare function merge-impl:build-final-properties(
             ($a, $b + map:entry("algorithm", $algorithm-info))
         },
         (),
-        if (merge-impl:properties-are-equal($instance-props, $docs)) then
```

Comment:
Unnecessary extra work is being done here. The standard merge accomplishes this and by leaving the logic in the standard merge, we don't restrict what a custom merge can do